### PR TITLE
feature(subscribe):Now user should be able to suscribe

### PR DESCRIPTION
**Pull request by Hodal66**
**What does this PR do?**

Implements subscriber option

**Description of Task to be completed?**

-User can subscribe, unsubscribe
-admin can see all subscribers

**How should this be manually tested?**

Clone the repo

checkout to branch __ft-subscriber-NewLetter__

Run `yarn install to install modules or **npm install modules**

Any background context you want to provide?
N/A

**What are the relevant Pivotal Tracker/Trello stories?**

[link to Trello Board]:: (https://trello.com/c/4I4sIejD/27-subscribe-backend)

__Screenshots (if appropriate)__
N/A

Questions:
N/A

### CODE REVIEW

File: controllers/subscriberController.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
+import Subscriber from "../model/subscribe.js";
+import { subscriberValidation } from "../validation.js";
+
+// export const allSubscribers = async (res) => {
```
Comment (PrinceNiyonshuti) [inline controllers/subscriberController.js:4]: always remember to remove the unused codes

**Review by PrinceNiyonshuti — CHANGES_REQUESTED** [review]
work on the feedbacks and you are ready to go

**Review by PatrickNiyogitare28 — CHANGES_REQUESTED** [review]
Just remove commented source codes as said above to be good to go. Remember to resolve the conflicts in this branch as well

**Review by shemalucien — CHANGES_REQUESTED** [review]
Remove unused codes

**Pull request merged**